File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1564_basico.js
import { question } from "readline-sync";

function main() {
    let reclamacao = get_Text();

    while (reclamacao !== null && reclamacao !== ``) {
        let numeroReclamacao = parseInt(reclamacao);

        if (numeroReclamacao === 0) {
            print(`vai ter copa!`);
        } else {
            print(`vai ter duas!`)
        }

        reclamacao = get_Text();
    }
}

function get_Text() {
    return question();
}

function print(text) {
    return console.log(text)
}

main();

File: Exercicios_ED_I/Bee/1564_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    int n;

    while (scanf("%d", &n) != EOF) {
        if (n == 0) {
            cout << "vai ter copa!" << endl;
        } else {
            cout << "vai ter duas!" << endl;
        }
    }
}